src/Microsoft.Health.Operations/OperationStatus.cs

```diff
@@ -39,4 +42,9 @@ public enum OperationStatus
     /// Specifies a status where the operation has stopped prematurely after being told to do so by the user.
     /// </summary>
     Canceled,
+
+    /// <summary>
+    /// Specifies a status where the operation has successfully completed execution.
+    /// </summary>
+    Succeeded,
```

Comment:
I have actually created a new value as it was impossible to serialize the value `"Completed"` for backwards compatibility.